### DISCUSSION TITLE
Rule to check that query param is missing

### DISCUSSION THREAD

**Discussion by mariantirlea** [mockoon/mockoon#385] [Help] [answered]
Hello everyone, 

I started to use Mockoon and I think it is a great tool. Thank you for this. ;)

I think I have to read more documentation but I am trying to understand if it is possible to create a rule that is triggered only when a query param is missing at all. Is this possible? I was only able to test that the value is empty.

```
/path1/path2?queryParam1=something
/path1/path2
```

I want to have a different response for the second example.

Thank you in advance,

**Comment by 255kb** [comment, marked as answer]
Hi @mariantirlea , thank you for using Mockoon 😄 

Currently, this is not possible, you can only test for an empty field with the appropriate regex but not for an absent field. There is an open issue: https://github.com/mockoon/mockoon/issues/255
Feel free to upvote it so we can prioritize.
